benchmark/infrastructure/BenchmarkMeasurementContainer.cpp

```diff
@@ -149,6 +148,17 @@ ResultTable::operator std::string() const {
   // to only combine them at the end.
   std::string prefix = absl::StrCat("Table '", descriptor_, "'\n");
 
+  // It's allowed to have tables without rows. In that case, we are already
+  // done, because there is no table content to print.
+  if (numRows() == 0) {
+    // In the case of no metadata, we don't need the line break at the end of
+    // the prefix.
+    return absl::StrCat(
+        prefix.substr(0, prefix.size() - 1),
+        addIndentation(getMetadataPrettyString(metadata(), "\nmetadata: ", ""),
+                       1));
+  }
+
```

Comment:
How does this look now for an empty table?
I would like something like the following (consistent with the "normal" formatting,
but explicitly tell us that there is no content, such that we know this.
```
metadata: { ... }
colName1 | colName2 | colName.... 
  ## Empty Table (0 rows) ##
```

Reply:
Done, see [6beb8f0](https://github.com/ad-freiburg/qlever/pull/964/commits/6beb8f04b748bd6771c97f567d4a70729058ac92).